src/test/java/kenigsberg/wordledictionary/WordleControllerTest.java

```diff
@@ -0,0 +1,81 @@
+package kenigsberg.wordledictionary;
+
+import org.junit.jupiter.api.Test;
+
+import javax.swing.*;
+import java.awt.*;
+
+import static org.mockito.Mockito.*;
+
+class WordleControllerTest {
+
+    //Create global variables of the mocks
+    WordleGame game = mock();
+    WordleDictionary dictionary = mock();
+    JLabel[][] labels = new JLabel[][]{
+            {mock(), mock(), mock(), mock(), mock()},
+            {mock(), mock(), mock(), mock(), mock()},
+            {mock(), mock(), mock(), mock(), mock()},
+            {mock(), mock(), mock(), mock(), mock()},
+            {mock(), mock(), mock(), mock(), mock()},
+            {mock(), mock(), mock(), mock(), mock()}
+    };
+
+
+    // given
+    WordleController controller = new WordleController(
+            game,
+            dictionary,
+            labels);
+
+    @Test
+    public void addLetter() {
+        // when
+        controller.addLetter("L");
+
+        // then
+        verify(labels[controller.guessNum][0]).setText("L");
+    }
+
+    @Test
+    public void enterGuess() {
```

Comment:
This test is failing when it's run on github because it's trying to use JOptionPane but it can't when it's running a test.

Reply:
How would I fix this issue?